new_core/performance.js

```diff
@@ -9,7 +9,11 @@ export function startMeasure (measureId) {
       lastDuration: 0,
       averageDuration: 0,
       lastTimestamp: -1,
-      fps: 0
+      count: 0,
```

Comment:
is count the same as number of iterations? if yes, then we should use the existing variable here.

Reply:
Count does count iterations, but it resets it every time a second is reached.

Reply:
isn't numberOfIterations % 1000 === 0 the same ? just try to figure out if we can prevent having two variables here

Reply:
That is a good solution

Reply:
Wait, I take it back. Count resets every time the frame duration is over 1000 milliseconds. That isn't the same as the numberOfIterations, because that number will continuously move forward forever. I can probably achieve this with a storage variable instead. It'll save us one line of code

measureData.jfps = measureData.numberOfIterations - measureData.count;
measureData.count = measureData.numberOfIterations;

Reply:
Didn't get rid of count, but did repurpose it to utilize the numberOfIterations instead. Just need a small persistent storage variable somewhere I can reach it. Closing this for now, start a new conversation if this solution is rough around the edges.